backend/src/services/mempool_monitor.py

```
@@ -1,224 +1,313 @@
 
-        except RPCException as e:
-            logger.error(f"RPC error during transaction monitoring: {e}")
-        except Exception as e:
-            logger.error(f"Error during transaction monitoring: {e}")
-        finally:
-            logger.info("Transaction monitoring stopped.")
-            await self._disconnect_websocket()
+        if is_new_token:
+            logger.info(f"Potential new token/pool transaction detected: {signature}")
+            # Run in background to not block the listener
+            asyncio.create_task(self._process_new_token_transaction(signature))
```

Ruff (0.15.10)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)